src/components/rich-text-widget.css

```diff
@@ -1,45 +1,9 @@
 .customRichTextWidget {
+  margin: 0;
+  border: 1px solid #999;
+}
 
-  :global .ccrte-modal-dialog {
-    margin: 0;
-
-    .dialog {
-      top: 0;
-      left: 50%;
-      transform: translate(-50%, 0);
-      z-index: 1;
-      .content {
-        .label {
-          padding: 0;
-          font-size: inherit;
-          font-weight: bold;
-        }
-        > .inline-row {
-          margin-top: -10px;
-          margin-bottom: -10px;
-        }
-        .checkbox {
-          input[type=checkbox] {
-            position: static;
-          }
-          input:checked + span::before {
-            font-family: arial;
-            font-size: 14px;
-            font-weight: bold;
-            top: 0.5rem;
-            left: 2.9rem;
-          }
-          span {
-            font-weight: bold;
-            margin-top: 0px;
-          }
-        }
-      }
-    }
-  }
-
-  .customRichTextEditor {
-    height: 100%;
-    overflow: hidden;
-  }
+.customRichTextEditor {
+  height: 100%;
+  overflow: hidden;
```

Comment:
The `.dialog` CSS here is for styling of dialogs put up by the `slate-editor` library. You should double-check that this change won't have adverse consequences in your use case.